packages/dso-toolkit/reference/render/modal-en-progress-indicator.html

```diff
@@ -0,0 +1,140 @@
+<header>
+  <div class="dso-logo-bar">
+    <img src="../../images/omgevingsloket.png" alt="Omgevingsloket" class="logo" />
+    <div class="dso-tools-bar">
+      <div class="dso-login">
+        <a href="" class="btn btn-link">
+          <dso-icon icon="user"></dso-icon><span >Inloggen</span>
+        </a>
+      </div>
+    </div>
+  </div>
+  <nav class="dso-navbar">
+    <div class="dso-navbar-header">
+      <button type="button" class="dso-navbar-toggle btn btn-default">
+        <dso-icon icon="bars"></dso-icon>
+        <span class="sr-only">Ga naar menu</span>
+      </button>
+    </div>
+    <ul class="dso-nav dso-nav-main">
+      <li>
+        <a href="#">
+          Home
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          Vergunningcheck
+        </a>
+      </li>
+      <li class="dso-active">
+        <a href="#">
+          Aanvragen
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          Regels op de kaart
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          Mijn Omgevingsloket
+        </a>
+      </li>
+    </ul>
+  </nav>
+  <nav aria-label="U bevindt zich hier:">
+    <ol class="breadcrumb" itemscope itemtype="//schema.org/BreadcrumbList">
+    </ol>
+  </nav>
+</header>
+<body class="dso-modal-open">
+  <div class="dso-modal" tabindex="-1" role="dialog" aria-labelledby="modal-dialog">
+    <div class="dso-dialog" role="document">
+      <div class="dso-body">
+        <div class="dso-progress-indicator dso-progress-indicator-looping dso-small" role="progressbar" aria-valuetext="Een moment geduld alstubliefts.">
+          <div class="dso-progress-indicator-spinner"></div><span class="dso-progress-indicator-label">Een moment geduld alstubliefts.</span>
```

Comment:
Een moment geduld alstubliefts? Komt dit wel uit de meest recente reference render, of was dit nog een typo van toen je zelf de markup had ingeklopt?